frontend/app/routes/public/renew/$id/ita/confirm-address.tsx

```diff
@@ -143,7 +146,7 @@ export default function RenewItaConfirmAddress() {
               errorMessage={errors?.hasAddressChanged}
               required
             />
-            {hasAddressChangedRadioValue && (
+            {!hasAddressChangedRadioValue && (
               <InputRadios
                 id="is-home-address-same-as-mailing-address"
```

Comment:
Since `hasAddressChangedRadioValue` is initially `undefined`, we don't want to render the radio buttons by default, but only if the user selects "No" to if their mailing address changed.

```suggestion
            {hasAddressChangedRadioValue === false && (
              <InputRadios
                id="is-home-address-same-as-mailing-address"
```